apps/i18n/common/en_us.json

```diff
@@ -1,5 +1,6 @@
 {
   "activity": "Activity",
+  "actions": "Actions",
   "addRemoveCleverClassrooms": "Visit https://clever.com/ and confirm your classroom is connected to Code.org.",
```

Comment:
nit: Can we re-use the "Actions" string that already exists? or keep your more generic "Actions" and swap it in for "catActions". Dupes might clutter the file.

Reply:
I don't want to use a specific feature string in an entirely different feature, so I made a general one. My understanding is that we are okay with having duplicates, because it allows us to go back and change feature specific strings.

Reply:
I _think_ it's also the case that as long as the English version is the same, you'll still get any existing translations)

Reply:
Oh that's cool